lib/Dialect/Torch/Transforms/FuseQuantizedOps.cpp

```diff
@@ -63,6 +63,14 @@ class QuantizeOperandsPastCommutingOps : public OpRewritePattern<SrcOp> {
     llvm::SmallVector<Value> operands(op->getOperands());
     bool dequanted = false;
 
+    // Prevent fusion for 1d convolution ops and just do it as an f32 conv since
+    // there isn't a linalg named op for quantized 1-d convolution yet.
```

Comment:
Yeah, this is good. Maybe add a `// TODO: Remove this and add support for 1-d quantized convolution.`

Also, I'd recommend changing the title to something like `[Torch] disable 1-d convolution quantization`, since this isn't really a "fix" for quantized 1-d convolution.